test/integration/helpers/serverspec/multi_spec.rb

```diff
@@ -11,16 +11,11 @@
   describe file("/etc/elasticsearch/#{vars['es_instance_name']}/elasticsearch.yml") do
     it { should be_file }
     it { should contain 'http.port: 9201' }
-    it { should contain 'transport.tcp.port: 9301' }
```

Comment:
I think this should now be `it { should contain 'transport.port: 9301' }` instead of being removed right?